packages/protoc-gen-es/README.md

```diff
@@ -59,83 +61,64 @@ PATH=$PATH:$(pwd)/node_modules/.bin \
   a.proto b.proto c.proto
 ```
 
-Note that we are adding `node_modules/.bin` to the `$PATH`, so that the protocol
-buffer compiler can find the plugin.
+Note that `node_modules/.bin` to the `$PATH` needs to be added to the page so that the Protobuf compiler can find the
```

Comment:
This sentence seems like it should be modified a bit?

Reply:
Thanks, modified a bit in https://github.com/bufbuild/protobuf-es/pull/926/commits/09f2fd130370ad2237475e266f65514b089956c6